Corona-Warn-App/src/main/res/layout/fragment_scan_check_in_qr_code.xml

```diff
@@ -4,8 +4,7 @@
     android:id="@+id/check_in_qr_code_scan_container"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:contentDescription="@string/qr_code_scan_title"
-    android:transitionName="shared_element_container">
```

Comment:
that is needed here, because of the transition from Scan FAB into this Screen

Reply:
whoops removed one to many of these :D should be back